packages/rich-list-api-client/__tests__/RichListApiClient.test.ts

```diff
@@ -0,0 +1,46 @@
+import nock from 'nock'
+import { RichListApiClient } from '../src/RichListApiClient'
+
+const client = new RichListApiClient({
+  url: 'http://rich-list-api-test.internal',
+  network: 'unit-test',
+  version: 'v0',
+  timeout: 60000
+})
+
+describe('RichListApiClient - get', () => {
+  it('Should be able to retrieve latest rich list', async () => {
+    nock('http://rich-list-api-test.internal')
+      .get('/v0/unit-test/foo')
+      .reply(200, function () {
+        return {
+          data: {
+            bar: ['1', '2']
+          }
+        }
+      })
+
+    const result = await client.requestData('GET', 'foo')
+    await expect(result).toStrictEqual({
+      bar: ['1', '2']
+    })
```

Comment:
Isn't this behaviour already tested in `RichListApiClient.spec.test.ts` ?  I think we don't need to test it again here.

Reply:
Oops, was some leftover from debugging